src/components/common/styles.js

```diff
@@ -163,64 +188,71 @@ export const StyledError = styled.div`
 
 export const StyledDiv = styled.div`
```

Comment:
More descriptive name

Reply:
For example, this seems to only be used in Landing.js atm.

Reply:
I changed it to StyledCenterContainer as it holds all the main components on the landing page in the center.